components/AllocatedCases/AllocatedCases.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import AllocatedCasesTable from 'components/AllocatedCases/AllocatedCasesTable';
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import { getWorkers, allocatedCases } from 'utils/api/allocatedWorkers';
+import Spinner from 'components/Spinner/Spinner';
+
+const AllocatedCases = ({ id }) => {
+  const [error, setError] = useState();
+  const [loading, setLoading] = useState();
+  const [allocWorkers, setAllocWorkers] = useState();
+  const [allocCases, setAllocCases] = useState();
+  const getData = useCallback(async () => {
+    try {
+      const workData = await getWorkers();
+      const casesData = await allocatedCases();
+      setAllocWorkers(workData);
+      setAllocCases(casesData);
+    } catch (e) {
+      setError(true);
+    }
+    setLoading(false);
+  });
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const worker = allocWorkers && allocWorkers.filter((team) => team.id == id);
```

Comment:
you can use `find` it's quicker (it stops when the first is found and it returns just the meaningful one)